splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -106,6 +107,14 @@ private void configureIngress() throws Exception {
             .end();
     }
 
+    private void configureSuccessHandler() throws Exception {
+        // If Event was sent successfully, send success reply to return kafka
+        from(direct("success"))
+            .setBody(simple("Response ${body}"))
```

Comment:
Why is the body formatted this specific way? What does Notifications expect?

Reply:
It was formatted this way to match what we are sending to logs below on line 157, just to keep the two responses consistent.  As for needs from Notifications, I think they just need some indication of success or failure along with which event is being referred to.  @pilhuhn is there anything specific you would prefer in the message when we return Success to return kafka?

Reply:
I was trying to test it locally (only integrations and kafka running in Docker), but I could not validate the response that is sent to kafka. I will setup the Ephemeral Env to try testing this

Reply:
@CodeHeeler I would avoid sending the whole body back. Maybe just an empty body or some quick log message with message id (if this suppose to be informational).

Reply:
`ce-id` would be uniqe CloudEvent ID, that seems like a great choice. Just, check whether it would work within simple when the name contains dash.